Documentation/operations/upgrade.rst

```diff
@@ -383,10 +383,16 @@ Added Metrics
 * ``cilium_node_health_connectivity_latency_seconds``
 * ``cilium_operator_unmanaged_pods``
 * ``cilium_policy_selector_match_count_max``
+* ``cilium_identity_cache_timer_duration``
+* ``cilium_identity_cache_timer_trigger_latency``
+* ``cilium_identity_cache_timer_trigger_folds``
 
 Removed Metrics
 ~~~~~~~~~~~~~~~
 * ``cilium_cidrgroup_translation_time_stats_seconds`` has been removed, as the measured code path no longer exists.
+* ``cilium_triggers_policy_update_total`` has been removed.
+* ``cilium_triggers_policy_update_folds`` has been removed.
+* ``cilium_triggers_policy_update_call_duration`` has been removed.
```

Comment:
These metrics are used in the [Cilium Grafana dashboard](https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/files/cilium-agent/dashboards/cilium-dashboard.json) included in the Helm chart. Could we get the dashboard updated too? Also, a sentence explaining why the metrics were removed might be helpful for people who used them despite being inaccurate.

Reply:
very good catch!  Do you mind if I fix that in a follow-up PR?

Reply:
yeah, a follow-up PR sounds good

Reply:
https://github.com/cilium/cilium/issues/36419